.gitignore

```diff
@@ -1,8 +1,10 @@
-**/target/
+target/
```

Comment:
won't that remove submodules’ targets from ignoring?

Reply:
No, the leading `**/` is optional, it has the same semantic as not being there. I rarely see it used in practice, other than in more complex cases like `/a/**/b`. 


https://git-scm.com/docs/gitignore

Reply:
TIL. Thanks!